tests/src/sanity.rs

```diff
@@ -88,6 +88,10 @@ mod tests {
             assert!(self.stderr.lock().unwrap().is_empty());
         }
 
+        fn assert_python_fileops_stderr(&self) {
+            assert!(self.stderr.lock().unwrap().contains("OK"));
```

Comment:
this is a very weak assertion.. we want to know mirrord layer didn't report errors as well. if tests passed by mirrord reached an unexpected scenario we want to fail :)